### ISSUE TITLE
 Most Recent Channel Videos in Feed 

### ISSUE THREAD

**Issue by aelsadi** [TeamPiped/Piped#3533]
### Describe the feature

An option that when selected, will allow only the most recent video and/or short from each channel to be shown in the feed. 

### Why would this be useful to add?

This feature will greatly clean up the homepage if the user is subscribed to channels that post very frequently.  Additionally, it may help users stay updated on channels that post less frequently, so they don't get hidden under more active channels. 

### Concept(s)

A checkbox in preferences that when selected, alters the feed to only show the most recent video and/or short (depending on the filter selected in the feed)  from each channel. 

### Acknowledgements

- [X] I have searched the existing issues and this is **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this form.

**Comment by matt-laird** [comment]
How is this different to "Hide watched videos in the feed" option + Sort by latest? Can you better describe what you mean by recent video? As what you described is already possible.

**Comment by aelsadi** [comment]
Ideally the feature would equalize the number of videos per channel displayed on the feed, by having each channel only display there lasted video shown. For example if I subscribed to 25 channels, my feed would have 25 videos, one from each channel.  

Regardless, I did find sort of a work around by splitting my feed into 2 subscription groups instead, one with the frequent channels and the other with the infrequent channels.  

Thanks for the suggestion though
